iptables/_rule.sls

```diff
@@ -11,6 +11,9 @@ iptables_{{ chain_name }}_{{ rule_name }}:
   {%- endif %}
   - table: {{ rule.get('table', 'filter') }}
   - chain: {{ chain_name }}
+  {%- if rule.family is defined %}
+  - family: {{ rule.family }}
```

Comment:
I thought about it, but I can't find a real-world example where the rule would be the same for both protocols. Also, it would complicate the code somewhat.